lapack.go

```diff
@@ -29,4 +29,35 @@ type Float64 interface {
 	Dpotrf(ul blas.Uplo, n int, a []float64, lda int) (ok bool)
 }
 
-type Complex128 interface{}
```

Comment:
Okay, thanks for checking. What do we mean by "support" in this package? I know we're not really sure, but support is pretty vague. My plan had been to add those functions needed by matrix, unless we explicitly need something.

Another possible option is to add all of the Go functions we have. However, there are a number of partial LAPACK implementations. Atlas and OpenBLAS both have a couple of functions, and I believe Eigen does as well. Keeping the interfaces minimal makes it easier for outside implementations to fully support them.